playbook/app/pb_kits/playbook/index.js

```diff
@@ -4,111 +4,7 @@ import 'lazysizes/plugins/attrchange/ls.attrchange'
-export { default as PbReactPopover } from './pb_popover/_popover'
-export { default as Person } from './pb_person/_person'
-export { default as PersonContact } from './pb_person_contact/_person_contact'
-export { default as PhoneNumberInput} from './pb_phone_number_input/_phone_number_input'
-export { default as Pill } from './pb_pill/_pill'
-export { default as ProgressPills } from './pb_progress_pills/_progress_pills'
-export { default as ProgressSimple } from './pb_progress_simple/_progress_simple'
-export { default as ProgressStep } from './pb_progress_step/_progress_step'
-export { default as ProgressStepItem } from './pb_progress_step/_progress_step_item'
-export { default as Radio } from './pb_radio/_radio'
-export { default as RichTextEditor } from './pb_rich_text_editor/_rich_text_editor'
-export { default as SectionSeparator } from './pb_section_separator/_section_separator'
-export { default as Select } from './pb_select/_select'
-export { default as SelectableCard } from './pb_selectable_card/_selectable_card'
-export { default as SelectableCardIcon } from './pb_selectable_card_icon/_selectable_card_icon'
-export { default as SelectableIcon } from './pb_selectable_icon/_selectable_icon'
-export { default as SelectableList } from './pb_selectable_list/_selectable_list'
-export { default as SelectableListItem } from './pb_selectable_list/_item'
-export { default as Source } from './pb_source/_source'
-export { default as StarRating } from './pb_star_rating/_star_rating'
-export { default as StatChange } from './pb_stat_change/_stat_change'
-export { default as StatValue } from './pb_stat_value/_stat_value'
-export { default as Table } from './pb_table/_table'
-export { default as TableRow } from './pb_table/_table_row'
-export { default as TextInput } from './pb_text_input/_text_input'
-export { default as Textarea } from './pb_textarea/_textarea'
-export { default as Time } from './pb_time/_time'
-export { default as TimeRangeInline } from './pb_time_range_inline/_time_range_inline'
-export { default as TimeStacked } from './pb_time_stacked/_time_stacked'
-export { default as Timeline } from './pb_timeline/_timeline'
-export { default as Timestamp } from './pb_timestamp/_timestamp'
-export { default as Title } from './pb_title/_title'
-export { default as TitleCount } from './pb_title_count/_title_count'
-export { default as TitleDetail } from './pb_title_detail/_title_detail'
-export { default as Toggle } from './pb_toggle/_toggle'
-export { default as Tooltip } from './pb_tooltip/_tooltip'
-export { default as TreemapChart } from './pb_treemap_chart/_treemap_chart'
-export { default as Typeahead } from './pb_typeahead/_typeahead'
-export { default as User } from './pb_user/_user'
-export { default as UserBadge } from './pb_user_badge/_user_badge'
-export { default as Walkthrough } from './pb_walkthrough/_walkthrough'
-export { default as WeekdayStacked } from './pb_weekday_stacked/_weekday_stacked'
+export * from './kits'
```

Comment:
If we do end up changing this file, we will need to update the generator. 
Right now, our kit generator (bin/rails g kit "NewKit") will throw the export into this file. 
This should definitely include an alpha as well. 
If possible, I'd leave this file the same, because of how much depends on it, but if we need to change it in order to do our testing, let's explore it a little. 

@markdoeswork 
cc: @nidaqg

Reply:
I use `Pathname.new("../app/pb_kits/playbook").children.select(&:directory?)` to get a list of all the folder names. I believe this to be the most reliable way to get a list of all the kits in `playbook/playbook`